jenkins/scripts/slurm_run.sh

```
@@ -106,28 +106,41 @@ pytest_exit_code=0
-    echo "DEBUG: Directory state at $llmSrcNode/tests/integration/defs:"
-    ls -l $llmSrcNode/tests/integration/defs
-
-    echo "DEBUG: conftest.py content:"
-    md5sum $llmSrcNode/tests/integration/defs/conftest.py
-
-    echo "DEBUG: pytest.ini content:"
-    md5sum $llmSrcNode/tests/integration/defs/pytest.ini
+# Use unified run_tests.py for test execution, rerun, and result merging
+runTestsScript="$llmSrcNode/jenkins/scripts/run_tests.py"
+perfModeFlag=""
+if [ "$perfMode" = "true" ]; then
```

Shellcheck (0.11.0)

[warning] perfMode is referenced but not assigned.

(SC2154)